build_packages.py

```diff
@@ -1,16 +1,49 @@
 import argparse
 import os
 from subprocess import check_call
+from pathlib import Path
+
+try:
+    from packaging.version import parse as Version, InvalidVersion
+except ImportError: # Should not happen, but at worst in most case this is the same
+    from pip._vendor.packaging.version import parse as Version, InvalidVersion
 
 DEFAULT_DESTINATION_FOLDER = os.path.join("..", "dist")
 package_list = ['azure-kusto-data', 'azure-kusto-ingest']
 
-
 def create_package(name, dest_folder=DEFAULT_DESTINATION_FOLDER):
     absdirpath = os.path.abspath(name)
     check_call(['python', 'setup.py', 'bdist_wheel', '-d', dest_folder], cwd=absdirpath)
     check_call(['python', 'setup.py', "sdist", '-d', dest_folder], cwd=absdirpath)
 
+def travis_build_package():
+    """Assumed called on Travis, to prepare a package to be deployed
+    This method prints on stdout for Travis.
+    Return is obj to pass to sys.exit() directly
+    """
+    travis_tag = os.environ.get('TRAVIS_TAG')
+    if not travis_tag:
+        print("TRAVIS_TAG environment variable is not present")
+        return "TRAVIS_TAG environment variable is not present"
+
+    try:
+        version = Version(travis_tag)
```

Comment:
This means your tag is a version, like "1.2.3", and you always release both of them at the same time correct?

Reply:
Yes